pages/news_items/news_items.ejs

```diff
@@ -33,6 +33,9 @@
                     <% if (news_item.unread) { %>
                     <span class="badge badge-danger ml-2">Unread</span>
                     <% } %>
+                    <% if (news_item.visible_to_students == true) { %>
+                    <span class="badge badge-secondary ml-2">Student</span>
+                    <% } %>
```

Comment:
Can we add this badge only if the person viewing the issues page is not a student?